tests/draft2019-09/ref.json

```diff
@@ -1048,5 +1048,43 @@
                  "valid": false
              }
          ]
-     }
+     },
+     {
+        "description": "$ref with $recursiveAnchor",
+        "schema": {
+            "$schema": "https://json-schema.org/draft/2019-09/schema",
+            "$id": "https://example.com/schemas/array-without-unevaluatedItems",
```

Comment:
I see what you're thinking, but because you have "unevaluatedItems" as the keyword, it reads differently.

Instead of saying "no unevaluatedItems", say "unevaluated items are disallowed".  Separating the keyword into two words and stating the implication more clearly makes all the difference here.